src/components/Alert.jsx

```diff
@@ -77,7 +77,7 @@ const Alert = ({
         )}
         <Button
           data-cy="alert-submit-button"
-          disabled={!isOpen}
```

Comment:
Ideally, the button should be disabled while an API call was in progress to prevent double submitting. This was disabling the button only during the transition state.